pkg/tfgen/installation_docs_test.go

```diff
@@ -304,6 +304,22 @@ func TestWriteInstallationInstructions(t *testing.T) {
 			repository:       "terraform-provider-testcase",
 			ghOrg:            "unicorncorp",
 		},
+		{
+			name: "Generates Deprecation Note If Provider on Deprecated List",
+			expected: autogold.Expect("~> **NOTE:** This provider has recently been deprecated by Pulumi.\n" +
+				"Going forward, it is available as a [Local Package](https://www.pulumi.com/blog/any-terraform-provider/) " +
+				"instead.\n" +
+				"Please see the [provider's repository](https://github.com/pulumi/pulumi-civo) for details.\n\n" +
```

Comment:
This note seems like it could be confusing if you're new to using civo with Pulumi; it's sounds like we're saying "don't use this it's deprecated!" 

Instead, I might suggest moving the note after the installation instructions and saying something along the lines of: "This provider was previously published as @pulumi/civo"; however that package is no longer being updated. Going forward, it is available as a Local Package instead. Please see the <repo name> for migration details."